External/FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -5724,7 +5724,7 @@ void OpDispatchBuilder::InstallHostSpecificOpcodeHandlers() {
   Initialized = true;
 }
 
-void InstallOpcodeHandlers(Context::OperatingMode Mode) {
+void InstallOpcodeHandlers(Context::OperatingMode Mode, Context::Context *CTX) {
```

Comment:
You don't want to do this in `InstallOpcodeHandlers`, you'll want to do this in `InstallHostSpecificOpcodeHandlers` just above this function.

Reply:
Oh, duh, missed that.